src/test/java/net/sf/jabref/logic/cleanup/CleanupWorkerTest.java

```diff
@@ -308,17 +308,6 @@ public void cleanupLatexMergesTwoLatexMathEnvironments() {
     }
 
     @Test
-    public void convertToBiblatexMovesJournalToJournalTitle() {
```

Comment:
I would leave this test here to make sure that the CleanupWorker works correctly wrt to the "Convert to BibLatex" option; the other test class for the BibLatexCleanupJob then only tests this class directly.